taxcalc/calculate.py

```diff
@@ -59,41 +59,28 @@ def calculator(params, records, mods="", **kwargs):
     while records.current_year < max_yr:
         records.increment_year()
 
-    params.update(final_mods)
+    params.implement_reform(final_mods)
     calc = Calculator(params, records)
     return calc
 
 
 class Calculator(object):
 
-    @classmethod
-    def from_files(cls, pfname, rfname, **kwargs):
-        """
-        Create a Calculator object from a Parameters JSON file and a
-        Records file
-
-        Parameters
-        ----------
-        pfname: filename for Parameters
-
-        rfname: filename for Records
-        """
-        params = Parameters.from_file(pfname, **kwargs)
-        recs = Records.from_file(rfname, **kwargs)
-        return cls(params, recs)
-
     def __init__(self, params=None, records=None, sync_years=True, **kwargs):
 
         if isinstance(params, Parameters):
             self._params = params
         else:
-            self._params = Parameters.from_file(params, **kwargs)
```

Comment:
It's asymmetric that it is permissable to create a Records object from a string which is a path to a file, but not a Parameters object from a string which is a path to a file. We don't have to use a `from_file` class factory method, but we should probably permit one to do so from the __init__. More detail in a later comment.